onmt/Models.py

```diff
@@ -113,55 +134,72 @@ def forward(self, src_input):
                                  .expand_as(emb))
             emb = self.dropout(emb)
 
-        out_length, out_batch, emb_size = emb.size()
+        out_length, out_batch, emb_dim = emb.size()
         aeq(in_length, out_length)
         aeq(in_length, out_length)
-        aeq(emb_size, self.embedding_size)
+        aeq(emb_dim, self.embedding_dim)
 
         return emb
 
+    @staticmethod
```

Comment:
There was a request that this not live on the class object. I think that is fine, can you make it either of function or part of a different "controller" type class.